scipy/stats/_distr_params.py

```diff
@@ -196,6 +198,7 @@
     ['cosine', ()],
     ['crystalball', (-1, 2)],
     ['dgamma', (-1, )],
+    ['dpareto_lognorm', (-3, 1.2, 1.5, 2)],
```

Comment:
This is the cause of the test failure. ($\mu = -3$ is valid.)
```suggestion
    ['dpareto_lognorm', (3, -1.2, 1.5, 2)],
```